draft-pda-protocol.md

```diff
@@ -670,17 +672,51 @@ collect request is said to be valid if all of the following conditions hold (let
 The leader responds to valid collect requests by first interacting with the
 helper as described in {{pa-aggregate}}. Once it has obtained the helper's
 encrypted output share for the batch, it responds to the collector's request
-with the following message:
+with HTTP status 200 and a body consisting of the following message:
 
 ~~~
 struct {
-  PPMTaskID task_id;
-  PPMProto proto;
-  PPMOutputShare leader_share;
-  opaque encrypted_helper_share;
+  PPMEncryptedOutputShare encrypted_leader_share;
+  PPMEncryptedOutputShare encrypted_helper_share;
```

Comment:
In `PPMReport` the encrypted input shares have type `PDAEncryptedInputShare encrypted_input_shares<1..2^16-1>`, which would allow PPM tasks to specify more than one helper. The current 
protocol doesn't support this right now, of course, but it might be a good idea to mirror what we have there.

```suggestion
  PPMEncryptedOutputShare encrypted_output_shares<0..2^16-1>;
```

If you take this suggestion, remember to update the text below.

Reply:
I'd prefer to punt this to #117, so that the document is updated in one step to handle multiple helpers.